### PR TITLE
Mdoc update xam12.10

### PR DESCRIPTION
**Pull request by lobrien**

### CODE REVIEW

**Comment by v-alje** [comment]
Docs Build status updates of commit _[4d5786f](https://github.com/xamarin/apple-api-docs/pull/140/commits/4d5786fd3fef0c51178bcb588df97ec4d1369764)_: 

:clock10: **Full build**: the average full build time is _46 min(s) 32 sec(s)_, this is based on the last _7_ build(s) for this repository.

 :warning: **This is a FULL build** due to a recent Docs Build service upgrade, normally no action is required on your part. In the result you find subsequent builds still performing full builds, please contact your repository admin to manually perform a full build on the 'master' branch. For more detailed info, please click [here](https://aka.ms/full-build-reasons).

**Comment by v-alje** [comment]
Docs Build status updates of commit _[4d5786f](https://github.com/xamarin/apple-api-docs/pull/140/commits/4d5786fd3fef0c51178bcb588df97ec4d1369764)_: 

:clock10: **Full build**: the average full build time is _46 min(s) 32 sec(s)_, this is based on the last _7_ build(s) for this repository.

 :warning: **This is a FULL build** due to a recent Docs Build service upgrade, normally no action is required on your part. In the result you find subsequent builds still performing full builds, please contact your repository admin to manually perform a full build on the 'master' branch. For more detailed info, please click [here](https://aka.ms/full-build-reasons).

**Comment by v-alje** [comment]
Docs Build status updates of commit _[4d5786f](https://github.com/xamarin/apple-api-docs/pull/140/commits/4d5786fd3fef0c51178bcb588df97ec4d1369764)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[en/ARKit/ARConfiguration.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/ARKit/ARConfiguration.xml) | :white_check_mark:Succeeded | |
[en/ARKit/ARFaceTrackingConfiguration.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/ARKit/ARFaceTrackingConfiguration.xml) | :white_check_mark:Succeeded | |
[en/ARKit/ARImageTrackingConfiguration.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/ARKit/ARImageTrackingConfiguration.xml) | :white_check_mark:Succeeded | |
[en/ARKit/ARObjectScanningConfiguration.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/ARKit/ARObjectScanningConfiguration.xml) | :white_check_mark:Succeeded | |
[en/ARKit/AROrientationTrackingConfiguration.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/ARKit/AROrientationTrackingConfiguration.xml) | :white_check_mark:Succeeded | |
[en/ARKit/ARSCNViewDelegate.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/ARKit/ARSCNViewDelegate.xml) | :white_check_mark:Succeeded | |
[en/ARKit/ARWorldTrackingConfiguration.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/ARKit/ARWorldTrackingConfiguration.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVAsset.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVAsset.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVAssetTrack.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVAssetTrack.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVAssetWriter.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVAssetWriter.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVAssetWriterInput.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVAssetWriterInput.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVAudioPlayer.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVAudioPlayer.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVAudioSession.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVAudioSession.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVAudioSessionDelegate.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVAudioSessionDelegate.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVAudioSessionPromptStyle.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVAudioSessionPromptStyle.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVAudioUnitComponent.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVAudioUnitComponent.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVCaptureAudioChannel.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVCaptureAudioChannel.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVCaptureAudioDataOutput.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVCaptureAudioDataOutput.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVCaptureAudioFileOutput.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVCaptureAudioFileOutput.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVCaptureAudioPreviewOutput.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVCaptureAudioPreviewOutput.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVCaptureConnection.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVCaptureConnection.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVCaptureDataOutputSynchronizer.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVCaptureDataOutputSynchronizer.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVCaptureDataOutputSynchronizerDelegate.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVCaptureDataOutputSynchronizerDelegate.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVCaptureDevice.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVCaptureDevice.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVCaptureDeviceFormat.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVCaptureDeviceFormat.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVCaptureDeviceInputSource.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVCaptureDeviceInputSource.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVCaptureDeviceTransportControlsPlaybackMode.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVCaptureDeviceTransportControlsPlaybackMode.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVCaptureFileOutput.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVCaptureFileOutput.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVCaptureFileOutputRecordingDelegate.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVCaptureFileOutputRecordingDelegate.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVCaptureFileOutputRecordingDelegate_Extensions.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVCaptureFileOutputRecordingDelegate_Extensions.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVCaptureMovieFileOutput.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVCaptureMovieFileOutput.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVCapturePhoto.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVCapturePhoto.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVCapturePhotoOutput.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVCapturePhotoOutput.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVCaptureScreenInput.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVCaptureScreenInput.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVCaptureSession.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVCaptureSession.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVCaptureVideoPreviewLayer.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVCaptureVideoPreviewLayer.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVContentAuthorizationStatus.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVContentAuthorizationStatus.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVContentKeyRequest.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVContentKeyRequest.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVContentKeySession.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVContentKeySession.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVContentKeySessionServerPlaybackContextOptions.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVContentKeySessionServerPlaybackContextOptions.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVErrorKeys.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVErrorKeys.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVFragmentedAsset.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVFragmentedAsset.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVFragmentedAssetTrack.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVFragmentedAssetTrack.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVMediaCharacteristic.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVMediaCharacteristic.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVMediaCharacteristics.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVMediaCharacteristics.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVMediaSelectionGroup.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVMediaSelectionGroup.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVMediaSelectionOption.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVMediaSelectionOption.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVMediaType.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVMediaType.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVMediaTypes.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVMediaTypes.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVMetadata.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVMetadata.xml) | :white_check_mark:Succeeded | |

This comment lists only the first 50 files in the pull request.
For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C23%5Ca3fa375f-32c5-dc35-e51a-d763709e81cb%5CPullRequest%5C201905230245022583-140%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=423Lwbj4klcFVyiYUZ30%2BDX5PRR41I0p7m%2B%2ByTugNnA%3D&st=2019-05-23T04%3A19%3A41Z&se=2019-06-23T04%3A24%3A41Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: en/AVFoundation/AVAssetTrack.xml [hunk 1]
```diff
@@ -246,34 +246,34 @@ public UIView (System.Drawing.RectangleF frame) : base (NSObjectFlag.Empty)
         <AssemblyVersion>0.0.0.0</AssemblyVersion>
       </AssemblyInfo>
       <Attributes>
-        <Attribute FrameworkAlternate="xamarin-macos-sdk-14">
+        <Attribute FrameworkAlternate="xamarin-macos-sdk-14;xamarin-mac-sdk-14">
```
Comment (conceptdev) [inline en/AVFoundation/AVAssetTrack.xml]: what's the difference between these two?

**Comment by conceptdev** [comment]
@lobrien what's with the two monikers?
`<Attribute FrameworkAlternate="xamarin-macos-sdk-14;xamarin-mac-sdk-14">`

**Comment by v-alje** [comment]
Docs Build status updates of commit _[8c7bae3](https://github.com/xamarin/apple-api-docs/pull/140/commits/8c7bae3d7e0d99265f4fbf6f362d3ac9abc36a8b)_: 

:clock10: **Full build**: the average full build time is _57 min(s) 11 sec(s)_, this is based on the last _8_ build(s) for this repository.

 :warning: **This is a FULL build** due to a recent Docs Build service upgrade, normally no action is required on your part. In the result you find subsequent builds still performing full builds, please contact your repository admin to manually perform a full build on the 'master' branch. For more detailed info, please click [here](https://aka.ms/full-build-reasons).

**Comment by lobrien** [comment]
The monikers were cruft from a misspelling. I didn't realize they were preserved over runs (although it makes sense in retrospect). The latest commits should fix.

**Comment by v-alje** [comment]
Docs Build status updates of commit _[8c7bae3](https://github.com/xamarin/apple-api-docs/pull/140/commits/8c7bae3d7e0d99265f4fbf6f362d3ac9abc36a8b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[en/ARKit/ARConfiguration.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/ARKit/ARConfiguration.xml) | :white_check_mark:Succeeded | |
[en/ARKit/ARFaceTrackingConfiguration.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/ARKit/ARFaceTrackingConfiguration.xml) | :white_check_mark:Succeeded | |
[en/ARKit/ARImageTrackingConfiguration.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/ARKit/ARImageTrackingConfiguration.xml) | :white_check_mark:Succeeded | |
[en/ARKit/ARObjectScanningConfiguration.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/ARKit/ARObjectScanningConfiguration.xml) | :white_check_mark:Succeeded | |
[en/ARKit/AROrientationTrackingConfiguration.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/ARKit/AROrientationTrackingConfiguration.xml) | :white_check_mark:Succeeded | |
[en/ARKit/ARSCNViewDelegate.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/ARKit/ARSCNViewDelegate.xml) | :white_check_mark:Succeeded | |
[en/ARKit/ARWorldTrackingConfiguration.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/ARKit/ARWorldTrackingConfiguration.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVAsset.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVAsset.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVAssetTrack.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVAssetTrack.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVAssetWriter.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVAssetWriter.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVAssetWriterInput.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVAssetWriterInput.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVAudioPlayer.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVAudioPlayer.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVAudioSession.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVAudioSession.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVAudioSessionDelegate.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVAudioSessionDelegate.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVAudioSessionPromptStyle.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVAudioSessionPromptStyle.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVAudioUnitComponent.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVAudioUnitComponent.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVCaptureAudioChannel.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVCaptureAudioChannel.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVCaptureAudioDataOutput.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVCaptureAudioDataOutput.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVCaptureAudioFileOutput.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVCaptureAudioFileOutput.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVCaptureAudioPreviewOutput.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVCaptureAudioPreviewOutput.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVCaptureConnection.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVCaptureConnection.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVCaptureDataOutputSynchronizer.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVCaptureDataOutputSynchronizer.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVCaptureDataOutputSynchronizerDelegate.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVCaptureDataOutputSynchronizerDelegate.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVCaptureDevice.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVCaptureDevice.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVCaptureDeviceFormat.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVCaptureDeviceFormat.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVCaptureDeviceInputSource.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVCaptureDeviceInputSource.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVCaptureDeviceTransportControlsPlaybackMode.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVCaptureDeviceTransportControlsPlaybackMode.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVCaptureFileOutput.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVCaptureFileOutput.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVCaptureFileOutputRecordingDelegate.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVCaptureFileOutputRecordingDelegate.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVCaptureFileOutputRecordingDelegate_Extensions.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVCaptureFileOutputRecordingDelegate_Extensions.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVCaptureMovieFileOutput.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVCaptureMovieFileOutput.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVCapturePhoto.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVCapturePhoto.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVCapturePhotoOutput.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVCapturePhotoOutput.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVCaptureScreenInput.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVCaptureScreenInput.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVCaptureSession.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVCaptureSession.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVCaptureVideoPreviewLayer.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVCaptureVideoPreviewLayer.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVContentAuthorizationStatus.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVContentAuthorizationStatus.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVContentKeyRequest.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVContentKeyRequest.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVContentKeySession.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVContentKeySession.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVContentKeySessionServerPlaybackContextOptions.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVContentKeySessionServerPlaybackContextOptions.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVErrorKeys.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVErrorKeys.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVFragmentedAsset.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVFragmentedAsset.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVFragmentedAssetTrack.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVFragmentedAssetTrack.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVMediaCharacteristic.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVMediaCharacteristic.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVMediaCharacteristics.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVMediaCharacteristics.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVMediaSelectionGroup.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVMediaSelectionGroup.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVMediaSelectionOption.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVMediaSelectionOption.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVMediaType.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVMediaType.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVMediaTypes.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVMediaTypes.xml) | :white_check_mark:Succeeded | |
[en/AVFoundation/AVMetadata.xml](https://github.com/xamarin/apple-api-docs/blob/mdoc_update_xam12.10/en/AVFoundation/AVMetadata.xml) | :white_check_mark:Succeeded | |

This comment lists only the first 50 files in the pull request.
For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C23%5Ca3fa375f-32c5-dc35-e51a-d763709e81cb%5CPullRequest%5C201905232048420631-140%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Wn5N%2B%2FA6j6ywWnq7TVQev3F%2BJEzGEqfQlm8%2BZ3KWO7M%3D&st=2019-05-23T22%3A01%3A44Z&se=2019-06-23T22%3A06%3A44Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by conceptdev** [comment]
@lobrien is this PR no longer required, given the newer PRs (and the merge conflicts here now)?

**Comment by lobrien** [comment]
Supplanted by 201906129.1

**Pull request closed without merging**